docs/running-e3sm-guide/guide-post-processing.md

```diff
@@ -0,0 +1,141 @@
+climatology plots averaged over years 21-50, ENSO plots for years 11 to 50, and time series plots covering years 1 to 50. The second realization will cover years 51-100 for climatologies, 11-100 for ENSO, and 1-100 for time series.
+
+`[global_time_series]`
+
+- `climo_years` and `ts_years` should match their equivalents in the `[mpas_analysis]` section.
+
+!!! tip
+    See the `zppy` [parameters documentation](https://e3sm-project.github.io/zppy/_build/html/main/parameters.html) for more information on parameters.
+
+### Launch zppy
+
+Run `zppy -c post.<case_name>.cfg`. This will submit a number of jobs. Run `sq` to see what jobs are running.
+
+`zppy` automatically handles dependencies of jobs. E.g., `e3sm_diags` jobs are dependent on `climo` and `ts` jobs, so they wait for those to finish. MPAS Analysis jobs re-use computations, so they are chained.
+
+Most jobs run quickly, though E3SM Diags may take around an hour and MPAS Analysis may take several hours.
+
+`zppy` creates a new directory `<simulations_dir>/<case_name>/post`. Each realization will have a shell script (typically `bash`). This is the actual file that has been submitted to the `batch` system. There will also be a log file `*.o<job ID>` as well as a `*.status` file. The status file indicates the state (WAITING, RUNNING, OK, ERROR). These files can be found in `<simulations_dir>/<case_name>/post/scripts`. Once all the jobs are complete, you can check their status.
+
+```shell
+cd <simulations_dir>/<case_name>/post/scripts
+cat *.status # should be a list of "OK"
+grep -v "OK" *.status # lists files without "OK"
+```
+
+If you re-run `zppy`, it will check the status of tasks and will skip a task if its status is “OK”. As your simulation progresses, you can update the post-processing years in the configuration file and re-run `zppy`. Newly added task will be submitted, while previously completed ones will be skipped.
+
+### Tasks
+
+If you run `ls <simulations_dir>/<case_name>/post/scripts` you’ll see files like `e3sm_diags_180x360_aave_model_vs_obs_0001-0020.status`. This is one e3sm_diags job. Parts of this file name are explained below:
+
+| Part of File Name | Meaning |
+| --- | --- |
+| `e3sm_diags` | Task |
+| `180x360_aave` | Grid |
+| `model_vs_obs` | `model_vs_model` or `model_vs_obs` |
+| `0001-0020` | First and last years |
+
+There is also a corresponding output file. It will have the same name but end with `.o<job ID>` instead of `.status`.
+
+### Output
+
+The post-processing output is organized hierarchically. Examples:
+
+- `<e3sm_simulations_dir>/<case_name>/post/atm/180x360_aave/ts/monthly/10yr` has the time series files – one variable per file, in 10 year periods as defined in `<run_scripts_dir>/post.<case_name>.cfg`.  
+- `<e3sm_simulations_dir>/<case_name>/post/atm/180x360_aave/clim/20yr` similarly has climatology files for 20 year periods, as defined in `<run_scripts_dir>/post.<case_name>.cfg``.
```

Comment:
Remove second backtick after `cfg` -- it's causing the code block to not render properly (including skipping the angled brackets)